frontend/typings.json

```diff
@@ -2,6 +2,7 @@
   "name": "lm-app",
   "dependencies": {},
   "globalDependencies": {
+    "node": "registry:dt/node#6.0.0+20160720070758",
```

Comment:
Why do yo need node?

If you want "require" intellisense and avoid `(require as any)` you can use [webpack-env](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/webpack/webpack-env.d.ts) typings